entries/n/ngpvan.com.json

```
@@ -0,0 +1,10 @@
+{
+  "Site Name": {
+    "domain": "ngpvan.com",
+    "img": "ngpvan.com.png",
+    "tfa": ["sms", "totp"],
+    "url": "https://myaccount.ngpvan.com/Account",
```

Because `CONTRIBUTING.md` says the domain should point to the main page, I think the `url` should be changed to ActionID's page.

```suggestion
    "url": "https://www.ngpvan.com/actionid/",
```